frontend/components/incompleteupload/Editor.vue

```diff
@@ -0,0 +1,49 @@
+<script setup lang="ts">
+import { type EditorIncompleteUpload } from '@/lib/editor'
+
+interface Props {
+  editorIncompleteUpload: EditorIncompleteUpload
+}
+interface Emits {
+  (e: 'update:editorIncompleteUpload', ei: EditorIncompleteUpload): void
+}
+const props = defineProps<Props>()
+const emit = defineEmits<Emits>()
+
+const model = computed({
+  get: () => props.editorIncompleteUpload,
+  set: (editorIncompleteUpload: EditorIncompleteUpload) => { emit('update:editorIncompleteUpload', editorIncompleteUpload) },
+})
+</script>
+
+<template>
+  <div>
+    <FormEditorField
+      help-text="The name of this uploaded source file."
```

Comment:
These strings don't seem particularly localizable, what's the plan there?

(also, a more general comment that I'm really happy on the `FormEditorField` paradigm you landed on, looks excellent here and elsewhere)